cli/README.md

```diff
@@ -13,75 +13,96 @@ mainflux-cli version
 ```
 
 ### User management
-#### Create User
+#### Create Users
 ```
 mainflux-cli users create john.doe@email.com password
 ```
 
-#### Login User
+#### Login Users
 ```
 mainflux-cli users token john.doe@email.com password
 ```
 
 ### System Provisioning
-#### Provision Device
+#### Create Devices
 ```
-mainflux-cli things create '{"type":"device", "name":"nyDevice"}' <user_auth_token>
+mainflux-cli things create '{"type":"device", "name":"myDevice"}' <user_auth_token>
 ```
 
-#### Provision Application
+#### Create Applications
 ```
-mainflux-cli things create '{"type":"app", "name":"nyDevice"}' <user_auth_token>
+mainflux-cli things create '{"type":"app", "name":"myDevice"}' <user_auth_token>
 ```
 
-#### Retrieve All Things
+#### Update Devices and Applications
+```
+mainflux-cli things update '{"id":"<thing_id>", "name":"myNewName"}' <user_auth_token>
+```
+
+#### Remove Things
+```
+mainflux-cli things delete <thing_id> <user_auth_token>
+```
+
+#### Retrieve all provisioned Things
 ```
 mainflux-cli things get all --offset=1 --limit=5 <user_auth_token>
 ```
 
-#### Retrieve Thing By ID
+#### Retrieve Things By ID
 ```
 mainflux-cli things get <thing_id> <user_auth_token>
 ```
 
-#### Remove Thing
+#### Create Channels
 ```
-mainflux-cli things delete <thing_id> <user_auth_token>
+mainflux-cli channels create '{"name":"myChannel"}' <user_auth_token>
 ```
 
-#### Provision Channel
-```
-mainflux-cli channels create '{"name":"nyChannel"}' <user_auth_token>
+#### Update Channels
 ```
+mainflux-cli channels update '{"id":"<channel_id>","name":"myNewName"}' <user_auth_token>
 
-#### Retrieve All Channels
 ```
-mainflux-cli channels get all --offset=1 --limit=5 <user_auth_token>
+#### Remove Channels
+```
+mainflux-cli channels delete <channel_id> <user_auth_token>
 ```
 
-#### Retrievie Channel By ID
+#### Retrieve all provisioned Channels
```

Comment:
I find this heading kinda misleading because the one cannot actually retrieve all the Channels, but a subset.